sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_v1/query.sql

```diff
@@ -113,78 +90,26 @@ events_summary AS (
       ),
       TRUE
     ) AS is_terminal,
-    COUNTIF(
-      res = 'default_partner_search_suggestion'
-    ) AS num_default_partner_search_suggestion_impressions,
-    COUNTIF(res = 'search_engine') AS num_search_engine_impressions,
-    COUNTIF(res = 'trending_suggestion') AS num_trending_suggestion_impressions,
-    COUNTIF(res = 'history') AS num_history_impressions,
-    COUNTIF(res = 'bookmark') AS num_bookmark_impressions,
-    COUNTIF(res = 'open_tabs') AS num_open_tab_impressions,
-    COUNTIF(res = 'admarketplace_sponsored') AS num_admarketplace_sponsored_impressions,
-    COUNTIF(res = 'navigational') AS num_navigational_impressions,
-    COUNTIF(res = 'add_on') AS num_add_on_impressions,
-    COUNTIF(res = 'wikipedia_enhanced') AS num_wikipedia_enhanced_impressions,
-    COUNTIF(res = 'wikipedia_dynamic') AS num_wikipedia_dynamic_impressions,
-    COUNTIF(res = 'weather') AS num_weather_impressions,
-    COUNTIF(res = 'quick_action') AS num_quick_action_impressions,
-    COUNTIF(res = 'pocket_collection') AS num_pocket_collection_impressions,
-    -- can't group by array-valued column. Instead "aggregate" by taking a single value
-    ANY_VALUE(experiments) AS experiments
+    CASE
+      WHEN  get_event_action(name, extra) IN ('engaged', 'annoyance')
+        THEN selected_result
+      ELSE NULL
+    END AS engaged_result_type,
+    CASE
+      WHEN get_event_action(name, extra) IN ('engaged', 'annoyance')
+        THEN product_selected_result
```

Comment:
similarly, this needs the mozfun.map.get_key expression with the product mapping applied